app/cmd.go

```diff
@@ -354,13 +354,13 @@ Migration: %s (#%d)
 				return errors.New("config file is required")
 			}
 
-			t, err := toml.LoadFile(file)
+			data, err := os.ReadFile(file)
 			if err != nil {
 				return err
 			}
 
 			var cfg remotemonitor.Config
-			err = t.Unmarshal(&cfg)
+			err = toml.Unmarshal(data, &cfg)
```

Comment:
How is it able to see `toml` now that the import has been changed?

Reply:
`toml` is the actual package name (i.e., in the `.go` files), so the prefix on the import line was redundant.

The change from `t` to `toml` here is due to the API change with v2 of the package.